test/libsolidity/syntaxTests/types/event_selector_type.sol

```diff
@@ -0,0 +1,20 @@
+event EvExt();
+error ErExt();
+
```

Comment:
Could also add a test case for file-level constants.
`bytes4 constant e = ErExt.selector;`